src/bptl/tasks/migrations/0009_auto_20201207_1814.py

```diff
@@ -0,0 +1,74 @@
+# Generated by Django 2.2.14 on 2020-12-07 17:14
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zgw_consumers", "0011_remove_service_extra"),
+        ("tasks", "0008_auto_20200228_1616"),
+        ("zgw", "0004_auto_20201207_1808"),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
```

Comment:
I didn't want to drop/create existing tables.

If you have the `CreateModel` or `RemoveModel` operations, they will create/drop the respective db tables. However, since I am moving the model from one app to another, I just want to rename the DB table and keep the existing data.

So, the migrations do that:

1. rename existing table to table name in new app
2. "remove" the model from the old app
3. "create" the model in the new app

Steps 2. and 3. are only from Django's model perspective, but have no actual changes in the database schema, since we handled that in step 1. already.